From 5199ade9ce4b09fdcc4bb4117d796eafe260cf56 Mon Sep 17 00:00:00 2001
From: Michael Orlitzky <michael@orlitzky.com>
Date: Tue, 14 Apr 2020 10:16:59 -0400
Subject: [PATCH 1/3] tests/MIP_Problem/Makefile.am: disable mipproblem2.

The "mipproblem2" test sometimes fails on PPC64 systems, and the
easiest way to work around that is to simply delete the test. Guess
what this commit does.

Gentoo-bug: https://bugs.gentoo.org/717258
Upstream-bug: https://www.cs.unipr.it/mantis/view.php?id=2653
---
 tests/MIP_Problem/Makefile.am | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/tests/MIP_Problem/Makefile.am b/tests/MIP_Problem/Makefile.am
index 80f66cbb2..911fca67d 100644
--- a/tests/MIP_Problem/Makefile.am
+++ b/tests/MIP_Problem/Makefile.am
@@ -55,7 +55,6 @@ TESTS = \
 ascii_dump_load1 \
 exceptions1 \
 mipproblem1 \
-mipproblem2 \
 mipproblem3 \
 mipproblem4
 
@@ -73,8 +72,6 @@ exceptions1_SOURCES = exceptions1.cc
 
 mipproblem1_SOURCES = mipproblem1.cc
 
-mipproblem2_SOURCES = mipproblem2.cc
-
 mipproblem3_SOURCES = mipproblem3.cc
 
 mipproblem4_SOURCES = mipproblem4.cc
-- 
2.24.1

